### DISCUSSION TITLE
Is there a way to hide an output of a code, similar to how code-fold works for code?

### DISCUSSION THREAD

**Discussion by jeweljohnsonj** [quarto-dev/quarto-cli#5131] [Q&A] [answered]
We have the option to hide/fold the code by using the following arguments in the code block:

```{r}
#| code-fold: true
```
I was wondering if there is a way to implement the same for the output of the code. Similar to how the code is hidden, this time, the output of the code will be hidden. 

I was making a tutorial showcasing different ways of writing a particular code, all of which gave the same output. I wanted to emphasize the differences in the code but at the same time give the readers the benefit of seeing that the output is the same. Therefore instead of using `eval: false` is there a way to implement a code-fold kind of feature for the output? Thanks!

**Comment by cderv** [comment, marked as answer]
This is already tracked in https://github.com/quarto-dev/quarto-cli/issues/341 and this is a duplicate of https://github.com/quarto-dev/quarto-cli/discussions/4911

I'll close this and you can follow the issue

**Comment by jeweljohnsonj** [comment]
Thank you.
